releasenotes/notes/add-dd_call_basic_config-env-15570281d176f438.yaml

```
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Add ``DD_CALL_BASIC_CONFIG={true,false}`` environment variable to control whether ``ddtrace`` calls ``logging.basicConfig``.
+    By default when using ``ddtrace-run`` or running in debug mode, we always call ``logging.basicConfig`` to ensure there is always a
+    root handler. This has compatibility issues for some logging configurations. With this new environment variable you can use
```

```suggestion
    root handler. This has compatibility issues for some logging configurations. ``DD_CALL_BASIC_CONFIG=false`` can be used to skip...
```

This is probably too messy to edit through github suggestions 😄